src/main/java/nextstep/optional/User.java

```diff
@@ -24,16 +26,16 @@ public boolean matchName(String name) {
     public static boolean ageIsInRange1(User user) {
         boolean isInRange = false;
 
-        if (user != null && user.getAge() != null
-                && (user.getAge() >= 30
-                && user.getAge() <= 45)) {
+        if (user != null && user.getAge() != null && (user.getAge() >= 30 && user.getAge() <= 45)) {
             isInRange = true;
         }
         return isInRange;
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .filter(u -> u.getAge() != null && u.getAge() >= 30 && u.getAge() <= 45)
```

Comment:
Nit: Stream의 `map()`을 활용해보셔도 좋을 것 같습니다.
```suggestion
                .map(User::getAge)
                .filter(age -> age >= 30 && age <= 45)
```

Reply:
넵!